### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by adr29truck**
**inspiration.md**
* Clarified

**README.md**
* Fixed typo
*  Added details about the new dark/light theme detection/automation in 0.114

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,8 @@
 OK so you've decided to redo your UI, don't worry, your UI will look as great as the screenshots in 5 simple steps! First of all, you will need to install [**card-mod**](https://github.com/thomasloven/lovelace-card-mod). It is a custom card available on [HACS](https://hacs.xyz) (the Home Assistant Community Store). Please read HACS's [documentation](https://hacs.xyz) and install it.
 
 ### 2. sun.sun
+As of version 0.114 of Home Assistant you are no longer required to use the ```sun.sun``` sensor to switch between dark and light mode **if** your device and browser support dark mode detection.
```
Comment (KTibow) [inline README.md:27]: Soft UI uses custom themes. Dark mode detection for custom themes isn't supported yet, so you should clarify this only works if you don't use a custom theme.

Comment (KTibow) [reply README.md:27]: Never mind, saw the part below. Cool feature! How about you link to that bottom part here?

Comment (adr29truck) [reply README.md]: Done

File: README.md [hunk 2]
```diff
@@ -35,8 +37,18 @@ This style works best with custom themes. You can also find and install them wit
 
 The themes used in the screenshots of this repo are the [Clear](https://github.com/naofireblade/clear-theme) and [Slate](https://github.com/seangreen2/slate_theme) theme by [**@naofireblade**](https://github.com/naofireblade) and [**@seangreen2**](https://github.com/seangreen2) (they are both available on HACS). If you decide to use the Clear theme, please make sure to remove the `lovelace-background` line from the theme YAML.
 
-### 4. Automation
-You will need to set up an automation to automatically switch to a dark theme at sunset and back to a light theme at sunrise. If you don't have one, please add the following to your `automations.yaml`.
+### <a name="automation"></a> 4. Automation
```
Comment (KTibow) [inline README.md:40]: This is unnecessary.
```suggestion
### 4. Automation 
```

File: README.md [hunk 3]
```diff
@@ -24,6 +24,8 @@
 OK so you've decided to redo your UI, don't worry, your UI will look as great as the screenshots in 5 simple steps! First of all, you will need to install [**card-mod**](https://github.com/thomasloven/lovelace-card-mod). It is a custom card available on [HACS](https://hacs.xyz) (the Home Assistant Community Store). Please read HACS's [documentation](https://hacs.xyz) and install it.
 
 ### 2. sun.sun
+As of version 0.114 of Home Assistant you are no longer required to use the ```sun.sun``` sensor to switch between dark and light mode **if** your device and browser support dark mode detection. See [4. Automation](#automation) for implementation.
```
Comment (KTibow) [inline README.md:27]: Change to
```suggestion
As of version 0.114 of Home Assistant you are no longer required to use the `sun.sun` sensor to switch between dark and light mode **if** your device and browser support dark mode detection. See [the automation section](#-4-automation) for implementation.
```

Comment (adr29truck) [reply README.md]: It appears as if this breaks the link.

File: README.md [hunk 4]
```diff
@@ -24,6 +24,8 @@
 OK so you've decided to redo your UI, don't worry, your UI will look as great as the screenshots in 5 simple steps! First of all, you will need to install [**card-mod**](https://github.com/thomasloven/lovelace-card-mod). It is a custom card available on [HACS](https://hacs.xyz) (the Home Assistant Community Store). Please read HACS's [documentation](https://hacs.xyz) and install it.
 
 ### 2. sun.sun
+As of version 0.114 of Home Assistant you are no longer required to use the `sun.sun` sensor to switch between dark and light mode **if** your device and browser support dark mode detection. See [the automation section](#.4-automation) for implementation.
```
Comment (KTibow) [inline README.md:27]: Another bug :(
```suggestion
As of version 0.114 of Home Assistant you are no longer required to use the `sun.sun` sensor to switch between dark and light mode **if** your device and browser support dark mode detection. See [the automation section](#4-automation) for implementation.
```

**Comment by N-l1** [comment]
Great. Thanks @adr29truck and @KTibow!

**Pull request merged**